utility/build-homebrew.sh

```diff
@@ -34,8 +36,10 @@ class Cumulusci < Formula
   homepage "https://github.com/SFDO-Tooling/CumulusCI"
   url $PACKAGE_URL
   sha256 $PACKAGE_SHA
-  head "https://github.com/SFDO-Tooling/CumulusCI.git"
+  license "BSD-3-Clause"
+  head "https://github.com/SFDO-Tooling/CumulusCI.git", branch: "main"
 
+  depends_on "rust" => :build # cryptography -> setuptools-rust
```

Comment:
```suggestion
```

Reply:
✍🏾 : The idea here is that we don't need this because we're installing this using pip, which makes use of binary wheels to avoid the need to compile dependencies like cffi/cryptography.